addons/mail/models/mail_template.py

```diff
@@ -60,7 +60,7 @@ def default_get(self, fields):
     mail_server_id = fields.Many2one('ir.mail_server', 'Outgoing Mail Server', readonly=False,
                                      help="Optional preferred server for outgoing mails. If not set, the highest "
                                           "priority one will be used.")
-    scheduled_date = fields.Char('Scheduled Date', help="If set, the queue manager will send the email after the date. If not set, the email will be send as soon as possible. Jinja2 placeholders may be used.")
+    scheduled_date = fields.Char('Scheduled Date', help="If set, the queue manager will send the email after the date. If not set, the email will be send as soon as possible. You can use inline_template expression.")
```

Comment:
I think "inline_template" does not mean anything. Just using "dynamic placeholders" or "dynamic expressions" should be sufficient, especially if we have a small docstring somewhere with what is supported (like in render mixin file as it is a main point outside of odoo core for rendering tools).